browser/ui/webui/brave_webui_source.cc

```diff
@@ -166,7 +166,6 @@ void CustomizeWebUIHTMLSource(const std::string &name,
         { "showFavoritesMenuLabel", IDS_BRAVE_NEW_TAB_SHOW_FAVORITES_MENU_LABEL },  // NOLINT
         { "showRewards", IDS_BRAVE_NEW_TAB_SHOW_REWARDS },
         { "showBinance", IDS_BRAVE_NEW_TAB_SHOW_BINANCE },
-        { "showTogether", IDS_BRAVE_NEW_TAB_SHOW_TOGETHER },
```

Comment:
This was never used; I think it may be from when we shared this preference on brave://settings/newTab